lib/devices/ledgerhid.js

```diff
@@ -331,6 +344,17 @@ class HIDDeviceInfo extends DeviceInfo {
     device.manufacturerName = device.manufacturer;
     return this.fromOptions(device);
   }
+
+  static isLedgerDevice(device) {
+    if (process.platform === 'win32' || process.platform === 'darwin') {
+      if (device.usagePage !== 0xffa0)
+        return false;
+    } else if (device.interface !== 0)
+      return false;
+
+    return device.vendorId === 0x2c97
```

Comment:
Do you know if this works with all the different types of ledgers?
Would it be useful to hold the hex numbers in some sort of constants object in case they change or new ones are added?